### DISCUSSION TITLE
Question about getting a test's output

### DISCUSSION THREAD

**Discussion by sezna** [mrcjkb/rustaceanvim#433] [Q&A] [answered]
Hi! Thank you for this awesome plugin.

When I run a test via testables, I get only the failure message, but not the output in a hover message. How would I go about getting the test output as well?

https://github.com/mrcjkb/rustaceanvim/assets/12157751/69d29a70-a20b-4507-8ec2-9d98bfa18fff

My config is on the left in the video, and the test is on the right. Relevant config pasted below:

```lua

require("lazy").setup({
...,
  {
    'mrcjkb/rustaceanvim',
    version = '^4', -- Recommended
    lazy = false, -- This plugin is already lazy
  },

... 
vim.keymap.set("n", "<leader>t", function() vim.cmd.RustLsp('testables') end, { desc = "(rust) see local testables" })
vim.keymap.set("n", "<leader>T", function() vim.cmd.RustLsp { 'testables', bang = true } end, { desc = "(rust) execute previous test command" })


```

**Comment by mrcjkb** [comment, marked as answer]
Hey :wave: 

In neovim 0.10, rustaceanvim defaults to the `background` executor for tests (see `:h rustaceanvim.config`).
I've just reverted it so that it defaults to the `termopen` executor (which I was planning to do soon anyway), which runs the tests in the terminal and displays all the output.

**Reply by sezna** [reply]
Thank you for the quick patch release! Updated and the new behavior is perfect.

**Reply by sezna** [reply]
One more quick question @mrcjkb: I would like the tests to default to a vert term instead of a horizontal term. do you know which config I would need to read to know how to do that?

Thanks!

**Reply by mrcjkb** [reply]
https://github.com/mrcjkb/rustaceanvim/discussions/337#discussioncomment-8963524 😃
